internal/transactionsubmission/transaction_worker.go

```
@@ -311,12 +380,23 @@ func (tw *TransactionWorker) reconcileSubmittedTransaction(ctx context.Context,
 		return fmt.Errorf("unexpected error: %w", hWrapperErr)
```

I guess we also need to report this error.